modules/mcp/config.py

```
@@ -0,0 +1,17 @@
+def can_build(env, platform):
```

Ruff (0.14.10)

Unused function argument: `env`

(ARG001)